org.sithlord48.BlackChocobo.yml

```diff
@@ -0,0 +1,58 @@
+app-id: org.sithlord48.BlackChocobo
```

Comment:
`sithlord48.org` doesn't exist. the app-id doesn't meet the requirements.

Reply:
will use `com.github.sithlord48.blackchocobo`